Parsers/ASimUserManagement/Parsers/vimUserManagementMicrosoftWindowsEvent.yaml

```diff
@@ -0,0 +1,242 @@
+Parser:
```

Comment:
can you please test filter for this parser from ASC portal?

Reply:
yes